bucket/rainbow.json

```diff
@@ -1,11 +1,14 @@
 {
-    "version": "1.0.1",
+    "version": "1.1.0",
     "description": "Print text in rainbow pattern.",
     "homepage": "https://github.com/arsham/rainbow",
     "license": "Apache-2.0",
-    "url": "https://github.com/arsham/rainbow/releases/download/v1.0.1/rainbow_windows_v1.0.1.zip",
-    "hash": "bab9af652642e1e7066b8c9c174a9621e9bed2e8acd1823001bd11b922488d24",
+    "url": "https://github.com/arsham/rainbow/releases/download/v1.1.0/figurine_windows_v1.1.0.zip",
+    "hash": "b7e37fb7b61c4ba2e34b3e67960ceda9a9253ebfeda3bf84add1361ad631b98c",
     "extract_dir": "deploy",
     "bin": "rainbow.exe",
-    "checkver": "github"
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/arsham/rainbow/releases/download/v$version/figurine_windows_v$version.zip"
```

Comment:
Can we add hash extraction for this?

The releases page contains Binary checksums

b7e37fb7b61c4ba2e34b3e67960ceda9a9253ebfeda3bf84add1361ad631b98c figurine_windows_v1.1.0.zip

Reply:
Yes I will do that